doc/modules/manifold.rst

```diff
@@ -417,7 +417,7 @@ version, the algorithms will try to preserve the order of the distances, and
 hence seek for a monotonic relationship between the distances in the embedded
 space and the similarities/dissimilarities.
 
-.. figure:: ../auto_examples/manifold/images/sphx_glr_plot_lle_digits_010.png
+.. figure:: ../auto_examples/manifold/images/sphx_glr_plot_lle_digits_002.png
```

Comment:
I do not think this is enough to fix `mainfold.rst` because of all the missing images. I think we need to rework https://github.com/scikit-learn/scikit-learn/pull/20543/ so that all the images are actually generated.

Basically https://github.com/scikit-learn/scikit-learn/pull/20543/ lead to a regression in `mainfold.rst`. The previous version was better: https://scikit-learn.org/0.24/modules/manifold.html#multi-dimensional-scaling-mds